Geometry/CMSCommonData/python/cmsExtendedGeometry2026D49XML_cfi.py

```diff
@@ -38,7 +38,7 @@
         'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker613/pixelStructureTopology.xml',
         'Geometry/TrackerSimData/data/PhaseII/TiltedTracker404/trackersens.xml',
         'Geometry/TrackerSimData/data/PhaseII/TiltedTracker404/pixelsens.xml',
-        'Geometry/TrackerRecoData/data/PhaseII/TiltedTracker613_MB_2019_04/trackerRecoMaterial.xml',
+        'Geometry/TrackerRecoData/data/PhaseII/OuterTracker616_2020_04/trackerRecoMaterial.xml',
```

Comment:
@mtosi I don't understand this change... 
Should not this file be instead moved to point to                  `Geometry/TrackerRecoData/data/PhaseII/TiltedTracker613_MB_2019_04/v2_ITonly/trackerRecoMaterial.xml`
as it was done for the customization PR: https://github.com/cms-sw/cmssw/pull/29910 ?